osu.Game/Screens/OnlinePlay/Multiplayer/MultiplayerPlayer.cs

```diff
@@ -25,6 +26,8 @@ public class MultiplayerPlayer : RoomSubmittingPlayer
     {
         protected override bool PauseOnFocusLost => false;
 
+        protected override UserActivity InitialActivity => new UserActivity.MultiplayerGame(Beatmap.Value.BeatmapInfo, Ruleset.Value);
```

Comment:
Similarly, this should probably live in `RoomSubmittingPlayer`.

Reply:
I think this is fine as-is since you aren't technically playing with others live compared to multiplayer.

Reply:
On the other side, it's not really "solo" either, is it? That's how it is right now.